### PR TITLE
Introduce bulk route api

### PR DESCRIPTION
**Pull request by kcudnik**
Purpose of this api is to speed up operations on routes, since we can have a lot of routes in system, and setting next hop for most of them can be done in one operation

### CODE REVIEW

**Comment by kcudnik** [comment]
Also to make it more generic we could use sai_object_key_t instead of sai_route_entry_t and we could use those declarations in other non object id objects

**Comment by kcudnik** [comment]
test this please

**Comment by kcudnik** [comment]
test this please

File: inc/saitypes.h [hunk 1]
```diff
@@ -594,11 +594,16 @@ typedef struct _sai_attribute_t
 
 typedef enum _sai_bulk_op_type_t
```
Comment (tushar-ty) [inline inc/saitypes.h]: This should be called 'mode' as 'type' is a bit misleading.

Comment (tushar-ty) [inline inc/saitypes.h]: To be more specific it should be called 'error_mode'.

Comment (kcudnik) [inline inc/saitypes.h]: this name is not subject of this PR, but sure error mode sounds better

Comment (tushar-ty) [inline inc/saitypes.h]: Yes, missed correcting this name earlier. But since we are at it, better to fix it before it gets much widely utilized.

Comment (kcudnik) [inline inc/saitypes.h]: sure, please fix your pr #560

Comment (tushar-ty) [inline inc/saitypes.h]: Done. Thanks. Pls approve the other one so that this can be fixed as well.

**Pull request merged**